torch/_utils.py

```diff
@@ -430,7 +430,7 @@ def reraise(self):
 
 
 def _get_available_device_type():
-    if torch.cuda.is_available():
+    if torch.cuda.device_count() > 0:
```

Comment:
Note: `is_available()` API cannot be compiled in JIT because it uses `hasattr` . Hence this is replaced by `device_count()`